docs/05-engineering/security-incidents.md

```diff
@@ -0,0 +1,49 @@
+# Security incidents
+
+Something went "bump" in the night (or the day)? This document explains what to do when responding to a security incident. See [What is an incident?](#what-is-an-incident) if you need help determining whether something counts as an incident.
+
+## Reporting phishing emails
+If you receive a phishing email, follow these steps to report to CivicActions IT:
+
+1. Do not click any links in the email. Do not delete it yet. You may mark it as spam.
+1. If you can, click the `Show Original` option in the "triangle" menu associated with the email. This will open a new window with the Original Message above and the raw text of the email below.
```

Comment:
Ah - this is gmail specific. I pulled the text from 18F where I gather they have standardized on gmail. We have people using many mail systems, so may have to get a bit more specific...